.gitignore

```diff
@@ -1,3 +1,4 @@
 .env
+.idea
```

Comment:
prefer using [a global gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html) for user-specific stuff (`.idea` has nothing to do with xpub-scan files)

Reply:
That is a good point. I will set up it locally